spec/blueberry_rails_spec.rb

```diff
@@ -1,65 +1,27 @@
 require 'minitest/autorun'
-require 'fileutils'
+require './spec/support/blueberry_rails_helpers.rb'
 
 class BlueberryRailsTest < Minitest::Unit::TestCase
+  include BlueberryRailsHelpers
 
   def setup
-    FileUtils.rm_rf 'test_project'
+    remove_project_directory
     cmd 'dropdb test_project_development'
     cmd 'dropdb test_project_test'
-
-    %w[RUBYOPT BUNDLE_PATH BUNDLE_BIN_PATH BUNDLE_GEMFILE].each do |key|
-      set_env key, nil
-    end
-  end
-
-  def teardown
-    restore_env
   end
 
   def test_rake_runs_cleanly
-    create_project 'test_project'
-    assert run_rake('test_project')
-  end
+    create_project
 
-  def test_rake_runs_cleanly_with_bootstrap_option
-    create_project 'test_project', '--bootstrap'
-    assert run_rake('test_project')
+    assert_exist_file 'config', 'initializers', 'simple_form.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.